addons/io/org.openhab.io.bin2json/README.md

```diff
@@ -44,7 +44,7 @@ final byte[] testdata = new byte[] {
 // @formatter:on
 
 JsonObject json = new Bin2Json(frameParserRule).convert(testdata);
-logger.debug(json.toString());
+logger.debug("{}", json.toString());
```

Comment:
Just for the future: this is actually more work for the runtime now. The argument replacement syntax is only useful for multiple arguments to avoid string operations if the debug level is not matched.